tei_adapter/tei_id_extractor/src/main/scala/weco/catalogue/tei/id_extractor/GitHubBlobContentReader.scala

```diff
@@ -0,0 +1,70 @@
+package weco.catalogue.tei.id_extractor
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.model.headers._
+import akka.http.scaladsl.unmarshalling.Unmarshal
+
+import java.net.URI
+import scala.concurrent.Future
+import scala.util.{Failure, Success, Try}
+import de.heikoseeberger.akkahttpcirce.ErrorAccumulatingCirceSupport._
+import io.circe.Decoder
+import uk.ac.wellcome.json.JsonUtil._
+import weco.http.client.HttpClient
+
+class GitHubBlobContentReader(httpClient: HttpClient, token: String)(
```

Comment:
Has this got any TEI logic in it? If not would it work to change the namespace here so it's not part of the id_extractor?

Reply:
No tei. It could technically live elsewhere although I'm not sure it's worth it right now